index.html

```diff
@@ -149,5 +174,9 @@ <h3 class="block-title">FAQ</h3>
       other Google APIs, the 2 libraries may be used by your application.</p>
       </div>
   </section> <!-- end of FAQ -->
+
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script>window.jQuery || document.write('<script src="_landing-page-static/js/vendor/jquery-1.10.2.min.js"><\/script>')</script>
```

Comment:
- This is roughly copy-pasta from `ruby` and `node`.
- They are both version `1.10.2`.
- The second only loads a script if `window.jQuery` is not present when the first completes
- The purpose of doing something like this is to use a CDN (ajax.googleapis.com) and fall back to self-hosted JS